src/main/scala/viper/gobra/frontend/info/implementation/typing/ghost/separation/GhostWellDef.scala

```diff
@@ -102,15 +102,6 @@ trait GhostWellDef { this: TypeInfoImpl =>
 
   private def exprGhostSeparation(expr: PExpression): Messages = expr match {
     case _: PGhostExpression => noMessages
```

Comment:
As a sanity check, is the following example rejected?

```
requires acc(x)
func foo(x *int) {
  *x = 0
}

ghost 
requires acc(x)
func bar(x *int) {
  ghost foo(x)
}
```

Reply:
It is and I've added it as another testcase. In particular, we have an assignment of a ghost variable to a non-ghost parameter and a call of a non-ghost method in a ghost context